deploy/templates/gcs-manifests/gcs-gd2-services.yml.j2

```diff
@@ -0,0 +1,64 @@
+---
+# Headless service for GD2 Stateful sets
+kind: Service
+apiVersion: v1
+metadata:
+  name: glusterd2
+  namespace: {{ gcs_namespace }}
+  labels:
+    gcs: glusterd2-service
+spec:
+  clusterIP: None
```

Comment:
Questions:
- Is there a reason to choose headless? (Do we need to reach the actual pod IP for some reason?)
- Is there a reason to have two very similar services (glusterd2 and glusterd2-client)?